pkgs/os-specific/linux/kernel/patches.nix

```diff
@@ -71,4 +71,17 @@
       hash = "sha256-DYPWgraXPNeFkjtuDYkFXHnCJ4yDewrukM2CCAqC2BE=";
     };
   };
+
+  rust-bindgen-version = rec {
```

Comment:
Can you add a comment where a good source for these patches is for posterity?

nit: The `rec` seems to be a copy'n'paste issue.